### PR TITLE
Compare Z-DWPC distributions

### PR DESCRIPTION
**Pull request by zietzm**
I think the [Kolmogorov–Smirnov test](https://en.wikipedia.org/wiki/Kolmogorov%E2%80%93Smirnov_test) is a good way to compare the distribution of observed (calculated) Z-DWPC values to the expected distribution of Z-values (the standard normal distribution). In the notebook is an implementation of this metric for the comparison of Z-DWPC scores.

### CODE REVIEW

**Comment by zietzm** [comment]
img-2616

In essence, this test will calculate the area of the shaded region, as a way to quantify how many more high Z-values we observe than we would expect.

File: hetmech/degree_group.py [hunk 1]
```diff
@@ -72,13 +70,11 @@ def compute_summary_metrics(df):
 
 def dwpc_to_degrees(graph, metapath, damping=0.5):
```
Comment (dhimmel) [inline hetmech/degree_group.py]: I like the new more efficient implementation (avoiding dense matrices which could cause memory problems). However, I think we want an option to yield zero-DWPC rows.

**Comment by dhimmel** [comment]
Do we need to track all of these files? I think it's too many to merge. What do we want to do?

**Comment by zietzm** [comment]
Outdated by the gamma hurdle model and newer pipeline code

**Pull request closed without merging**